src/phpdoc/parser.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/z7zmey/php-parser/pkg/token"
 )
 
+type PhpDoc struct {
```

Comment:
In Go, it's idiomatic to avoid stutter where possible.
Package name is a part of the type prefix, so you can call this struct `Comment`.
```diff
- phpdoc.PhpDoc
+ phpdoc.Comment
```